api/integrations/launch_darkly/launch_darkly.py

```diff
@@ -0,0 +1,239 @@
+            )
+
+        environments = []
+        self.logger.info(f"Adding {len(ld_environments)} environments")
+        for ld_environment in ld_environments:
+            environment = self._create_environment(ld_environment, project)
+            environments.append(environment)
+
+        self.logger.info(f"Adding {len(ld_flags)} features")
+        for ld_flag in ld_flags:
+            self._create_feature(ld_flag, environments, project)
+
+        self.logger.completed_at(datetime.now())
+        self.logger.completed = True
+        self.logger.save()
+
+        self.logger.info("Finished importing")
+
+    def _create_project(self, organisation_id: UUID, ld_project: dict) -> Project:
+        return Project.objects.create(
+            organisation_id=organisation_id, name=ld_project.get("name")
+        )
+
+    def _create_environment(self, ld_environment, project: Project) -> Environment:
+        return Environment.objects.create(
+            name=ld_environment.get("name"),
+            project=project,
+        )
+
+    def _create_feature(
+        self, ld_flag, environments: list[Environment], project: Project
+    ):
+        match ld_flag.get("kind"):
+            case "boolean":
+                self._create_boolean_feature(ld_flag, environments, project)
+            case "multivariate":
+                self._create_multivariate_feature(ld_flag, environments, project)
+            case _:
+                raise Exception("Invalid flag type from Launch Darkly")
+
+    def _create_boolean_feature(
+        self, ld_flag, environments: list[Environment], project: Project
+    ):
+        # todo try get tags
+        try:
+            feature = Feature.objects.create(
+                name=ld_flag.get("key"),
+                project=project,
+                initial_value=None,  # todo
+                description=ld_flag.get("description"),
+                default_enabled=None,  # todo
+                type=STANDARD,
+                tags=None,  # todo
+                is_archived=ld_flag.get("archived", False),
+                owners=None,  # todo
+                is_server_key_only=None,  # todo
+            )
+        except IntegrityError:
+            # Feature with this name already exists
+            self.logger.error(
+                f"Unable to create feature with name: {ld_flag.get('key')}"
+            )
+
+        for environment in environments:
+            ld_environment = ld_flag.get("environments", {}).get(environment.name)
+            if ld_environment:
+                feature_state = FeatureState.objects.create(
+                    feature=feature,
+                    environment=environment,
+                    identity=None,  # todo
+                    feature_segment=None,  # todo
+                    enabled=ld_environment.get("_summary").get("on"),
+                )
+                FeatureStateValue.objects.create(feature_state=feature_state)
```

Comment:
I'm surprised this works tbh, by creating the feature above, it should create the FeatureState in all environments. We probably need to do something like: 

```python
            feature = Feature.objects.create(
                ...
                default_enabled=False,  # default to false and set per environment based on LD values
                ...
            )
            for environment in environments:
                 ld_environment = ld_flag.get("environments", {}).get(environment.name)
                 if ld_environment:
                      feature_state = FeatureState.objects.filter(
                          feature=feature,
                          environment=environment,
                          identity=None,  # todo
                          feature_segment=None,  # todo
                      ).update(enabled=ld_environment.get("_summary").get("on"))
```